src/main/java/edu/ucsb/cs156/example/entities/UCSBOrganization.java

```diff
@@ -0,0 +1,22 @@
+package edu.ucsb.cs156.example.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "ucsborganizations")
+public class UCSBOrganization {
+    @Id
```

Comment:
Curious, what does each instance of UCSBOrganization have for its ID? Is it a string or a number?

Reply:
string!